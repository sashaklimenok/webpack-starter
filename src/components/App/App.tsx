import React, { ReactElement } from 'react';
import ava from 'assets/img/avatar.png';

export const App = (): ReactElement => {
  return (
    <>
      <img src={ ava } alt="ava"/>
      <h1>Hello World</h1>
    </>
  );
};
