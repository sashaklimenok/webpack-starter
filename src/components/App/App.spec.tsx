import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { App } from './App';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });


describe('Test', () => {
  it('should return 10', () => {
    const number = 10;
    const wrapper = shallow(<App />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
    expect(number).toBe(10);
  });
});
