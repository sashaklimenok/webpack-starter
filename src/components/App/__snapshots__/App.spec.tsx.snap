// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test should return 10 1`] = `
<Fragment>
  <h1>
    Hello World!
  </h1>
</Fragment>
`;
